CMakeLists.txt

```
@@ -0,0 +1,18 @@
+cmake_minimum_required(VERSION 3.9 FATAL_ERROR)
+
+# Set project name
+project(
+  "fortran-tooling"
+  LANGUAGES "Fortran"
+  VERSION "0.0.1"
+  DESCRIPTION "Fortran tooling"
+)
+
```

My gfortran is again unhappy about line truncation. This fixes it for gfortran but I'm not sure if this is portable
```suggestion
set(CMAKE_Fortran_FLAGS "-ffree-line-length-none")
```